src/main/flight/pid_init.c

```diff
@@ -549,7 +549,12 @@ void pidInitConfig(const pidProfile_t *pidProfile)
     pidRuntime.feedforwardTransition = pidProfile->feedforward_transition / 100.0f;
     pidRuntime.feedforwardTransitionInv = (pidProfile->feedforward_transition == 0) ? 0.0f : 100.0f / pidProfile->feedforward_transition;
     pidRuntime.feedforwardAveraging = pidProfile->feedforward_averaging;
-    pidRuntime.feedforwardSmoothFactor = 1.0f - (0.01f * pidProfile->feedforward_smooth_factor);
+    // feedforward_smooth_factor effect previously would change based on packet looprate
+    // normalizing to 250hz packet rate as that is the most commonly used ELRS packet rate
+    float scaledSmoothFactor = 0.01f * pidProfile->feedforward_smooth_factor;
+    float rxDt = 1.0f / 250.0f;
+    float feedforwardSmoothingTau = (rxDt * scaledSmoothFactor) / (1.0f - scaledSmoothFactor);
+    pidRuntime.feedforwardSmoothFactor = feedforwardSmoothingTau;
```

Comment:
Why do we re-calculate this `scaledSmoothFactor` value every loop, when it is constant?  
`rxDt` is constant (always 1.250)
`scaledSmoothFactor` is determined by the PID Profile value.
So... if we want to have a 'constant' pidRuntime value for feedforwardSmoothFactor, we should calculate that in pid_init.c, like we do now.

Reply:
This is in the pid_init.c file. This is only calculated once.